airflow/settings.py

```diff
@@ -516,6 +522,7 @@ def _session_maker(_engine):
             )
 
     Session = scoped_session(_session_maker(engine))
+    NonScopedSession = _session_maker(engine)
```

Comment:
I think actually all I want is the Session global to reuse this session maker object - so that we have just one `_session_maker()` "in scope". Does it work if you do this
```suggestion
NonScopedSession = _session_maker(engine)
     Session = scoped_session(NonScopedSession)
    
```

Reply:
let me try that

Reply:
That I think will work. What is not working is the thread local registry.

Reply:
This works, done.